### ISSUE TITLE
Width-full no longer full width after Gutenberg 13.7.1

### ISSUE THREAD

**Issue by kauaicreative** [WordPress/gutenberg#42604]
This is an issue again after the update Gutenberg 13.7.1

With Gutenberg on left, without on right.

<img width="2560" alt="image" src="https://user-images.githubusercontent.com/127033/180227612-d13737d5-d345-452e-ab03-7b3b6a4aa414.png">

_Originally posted by @kauaicreative in https://github.com/WordPress/gutenberg/issues/42374#issuecomment-1191499068_

**Comment by Mamaduka** [comment]
Thanks for contributing, @kauaicreative.

Can you provide steps for reproduction? I just tested with Image block, and full width works as expected.

**Comment by kauaicreative** [comment]
@Mamaduka Images have the same issue for me. 
Chrome, Mac 
Wordpress 6.0.1
Gutenberg 13.7.1
Twenty Twenty-TwoVersion: 1.2

<img width="1795" alt="image" src="https://user-images.githubusercontent.com/127033/180256993-4a647232-bf18-4102-9241-87e6ee237f56.png">

**Comment by kauaicreative** [comment]
@Mamaduka Note that the issue only appears on the front end. the editor view is never full width.

**Comment by kathrynwp** [comment]
@Mamaduka  Thanks for having a look.

### To replicate:

- WordPress 6.0.1
- Activate Twenty Twenty-Two 1.2
- Activate Gutenberg 13.7.0 (have not tested with other versions of GB)
- Add a Cover block to a static page using the default template
- Set the Cover block to full width
- Look at the page on the front end

### Result

With Gutenberg 13.7.0 active, the block is **not** full width

![Self-hosted_Test_–_not_just_another_fake_company](https://user-images.githubusercontent.com/7662449/180257414-0670e406-e78d-42a0-86ff-517d65d74346.jpg)

### Expected

The block should be full-width on the front end. With Gutenberg **deactivated**, it displays as full-width as expected:

![Self-hosted_Test_–_not_just_another_fake_company](https://user-images.githubusercontent.com/7662449/180257543-10acfcae-ccc8-4c70-afc7-207577c487bd.jpg)

I was **not** able to replicate with Twenty Twenty-One; the full-width cover block remains full width even with Gutenberg active. So this might be related to FSE/block themes.

This issue was also reported here: https://wordpress.org/support/topic/unexpected-lef-and-right-margins/

**Comment by kauaicreative** [comment]
@kathrynwp It's of interest that, with or without Gutenberg active, the editor always displays left and right margins. It could be argued that the front end showing 100% full width without Gutenberg is the bug. With that said, having 100% full width in a design seems important.

**Comment by kauaicreative** [comment]
e.g.  With Gutenberg on left, without on right. Front end on bottom.
(Also, note that Gutenberg is removing the bottom margin on the group block in the editor.)

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/127033/180261497-6ed5f6a5-5060-447c-9d2d-1cb17175079b.png">

**Comment by bgardner** [comment]
I can confirm this issue, as I noticed the same thing on our Frost website.

**Pre-update to Gutenberg 13.7.1:**

![image](https://user-images.githubusercontent.com/486261/180261916-67f81237-6537-4711-b2d5-f7f927062f0a.png)


**Post-update to Gutenberg 13.7.1:**

![image](https://user-images.githubusercontent.com/486261/180261936-68118b1e-0646-4dda-897d-2f8d43e5bdfc.png)

**Comment by kathrynwp** [comment]
@kauaicreative I think in general full-width blocks are intended to be displayed full-width, both in the editor and on the front end. Thanks for the additional screenshots. :)

**Comment by tellthemachines** [comment]
Seems that an overly-specific CSS rule coming from Core is overriding the negative margins defined by the theme:

<img width="308" alt="Screen Shot 2022-07-22 at 9 19 32 am" src="https://user-images.githubusercontent.com/8096000/180331342-e85173af-c54c-4d3d-a5ac-fa7ad3de1ab8.png">

The issue has since been fixed on trunk, but the fix hasn't been released yet.

**Comment by andrewserong** [comment]
@tellthemachines and I have merged a fix for this in #42614 to the `release/13.7` branch. It pulls in the fix from #42085 which is due to land in `13.8` (without all the features from the rest of that PR).

We've tagged and pushed a `13.7.2` point release here that includes the fix: https://github.com/WordPress/gutenberg/releases/tag/v13.7.2

I'm just about to wrap up for my week, but let us know if there is more feedback on this issue and we can factor it in to further layout changes in the next release.
